src/index.html

```diff
@@ -11,7 +11,174 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <!-- <h1>HTML Form</h1> -->
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+
+      <fieldset class="selection">
+        <legend>Personal information</legend>
+
+        <label class="formfield">
+          Surname:
+          <input
+            type="text"
+            name="Surname"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="formfield">
+          Name:
+          <input
+            type="text"
+            name="Name"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="formfield">
+          How old are You?
+          <input
+            type="number"
+            min="1"
+            max="100"
+            value="12"
+            name="Age"
+            required
+          >
+        </label>
+
+        <label class="formfield">
+          Full date of birth:
+          <input
+            type="date"
+            name="Full day or birth"
+          >
+        </label>
+
+        <label class="formfield">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="accept"
+          >
+        </label>
+
+      </fieldset>
+
+      <fieldset class="selection">
+        <legend>Registration</legend>
+
+        <label class="formfield">
+          E-mail:
+          <input
+            type="email"
+            name="E-mail"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+
+        <label class="formfield">
+          Password:
+          <input
+            type="password"
+            name="Password"
+            required
+          >
+        </label>
+
+      </fieldset>
+
+
+      <fieldset class="selection">
+        <legend>An interesting fact about you!</legend>
+
+        <label class="formfield">
+          Do you love cats?
+          <label >
+            <input
+              type="radio"
+              name="cat"
+              value="m"
+            >Yes
+          </label>
+
+          <label >
+            <input
+              type="radio"
+              name="cat"
+              value="f"
+            >No
+          </label>
```

Comment:
Because it's entirely wrapped in label when I click on any place on this row it always selects "Yes" option. Fix this.
```suggestion
          <label >
            <input
              type="radio"
              name="cat"
              value="m"
            >
            Yes
          </label>

          <label >
            <input
              type="radio"
              name="cat"
              value="f"
            >
            No
          </label>
```